tasks/webhook

```diff
@@ -74,9 +74,9 @@ def distributed_queue(amqp_server):
         logging.error('Please format amqp_server as host:port')
         sys.exit(1)
 
-    context = ssl.create_default_context(cafile='/run/secrets/webhook/ca.pem')
-    context.load_cert_chain(keyfile='/run/secrets/webhook/amqp-client.key',
-                            certfile='/run/secrets/webhook/amqp-client.pem')
+    context = ssl.create_default_context(cafile=os.path.join(WEBHOOK_SECRETS, 'ca.pem'))
```

Comment:
lis's list of favourite path mangling technologies (in order):
 - pathlib.Path
 - f-strings
 - ...
 - os.path.join()

Not worth getting into a fight over, though ;)

Reply:
Path() is too much overhead for my taste, so I picked your second favorite :grin: